get files from other branch 'update_modelsec4_bis'

loumir opened the pull request:
in Mango.tex intro to Model description section 
subsection for "building blocks" : 
     include file model_principles_head.tex
subsection for model dictionary ( by alphabetic order )
    include model.tex ( automatically generated from xml)

loumir committed:
get files from other branch 'update_modelsec4_bis'

lmichel commented on doc/MANGO.tex:
```diff
@@ -317,6 +319,14 @@ \subsubsection{MANGO and MIVOT: Structuring Tabular Data}
+
+\subsection{Building blocks}
+  \input{model_principles_head.tex}
+
```
this text is part of the generated descriptions, it cannot be out of model.tex

lmichel commented on doc/model_principles_head.tex:
```diff
@@ -0,0 +1,41 @@
+%---------------------------------------
+ 
+ The main class MangoObject represents the astronomical source or detection.
+ It is connected to a set of Properties observed and served by the catalog . 
```
or computed

loumir committed:
Update MANGO.tex

section 4 back to previous version . 
more thoughts to give on the presenttaion of building blocks . 

other proposer typo corrections left unchanged

loumir committed:
Update MANGO.tex

loumir reviewed (commented):
this PR is given up . 
more thoughts needed for the building block section .
